x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/view_results_panel/view_results_panel.tsx

```diff
@@ -7,20 +7,26 @@
 import React, { FC, Fragment } from 'react';
 import { EuiCard, EuiIcon } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { useNavigateToPath } from '../../../../../contexts/kibana';
-import { getResultsUrl } from '../../../analytics_management/components/analytics_list/common';
+import { useMlUrlGenerator } from '../../../../../contexts/kibana';
 import { ANALYSIS_CONFIG_TYPE } from '../../../../common/analytics';
+import { useNavigateToPath } from '../../../../../contexts/kibana';
+import { ML_PAGES } from '../../../../../../url_generator';
 
 interface Props {
   jobId: string;
   analysisType: ANALYSIS_CONFIG_TYPE;
 }
 
 export const ViewResultsPanel: FC<Props> = ({ jobId, analysisType }) => {
+  const urlGenerator = useMlUrlGenerator();
   const navigateToPath = useNavigateToPath();
 
-  const redirectToAnalyticsManagementPage = async () => {
-    const path = getResultsUrl(jobId, analysisType);
+  const redirectToAnalyticsJobPage = async () => {
+    const path = await urlGenerator.createUrl({
```

Comment:
When I click the 'View results' card at the end of the DFA wizard, the link it is trying to open has a duplicate base path `qdl/app/ml`:

```
http://localhost:5601/qdl/app/ml/qdl/app/ml/data_frame_analytics/exploration?_g=(ml%3A(analysisType%3Aclassification%2CjobId%3Amushroom_classification_clone))
```

Reply:
Thanks for spotting this. There was an issue for urls with base path. The fix is here https://github.com/elastic/kibana/pull/75696/commits/8179168feab07b3086bebd638cb14ef54811d297